JavaScript/chapter02/p01_remove_dups/avc278.js

```diff
@@ -0,0 +1,110 @@
+// Remove Dups: Write code to remove duplicates from an unsorted linked list.
+// Follow Up: How would you solve this problem if a temporary buffer is not allowed?
+
+const assert = require("assert");
+
+class LinkedListNode {
+  constructor(val, next) {
+    this.val = val === undefined ? null : val;
```

Comment:
optional: there's a [snazzy new way](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) to express this!

```suggestion
    this.val = val ?? null;
```

Reply:
This is definitely going to take some time to get used to.
But it's definitely superior for code gold.